components/error/errorCardMobile.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div

```

Comment:
Maybe use the Vuetify `VCard` component in order to have consistent styling with the rest of the application.